src/Components/Authentication/Authentication.js

```diff
@@ -7,16 +7,16 @@ import Unavailable from '@redhat-cloud-services/frontend-components/Unavailable'
 import propTypes from 'prop-types';
 import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
 import NotAuthorized from '@redhat-cloud-services/frontend-components/NotAuthorized';
+import { usePermissionsWithContext } from '@redhat-cloud-services/frontend-components-utilities/RBACHook';
 
 const Authentication = ({ children }) => {
   const queryClient = useQueryClient();
   const location = useLocation();
   const chrome = useChrome();
-  const { isLoading, isFetching, isSuccess, isError, data } = useUser();
-  const hasAnyPermission =
-    data?.rbacPermissions &&
-    (data.rbacPermissions.canReadActivationKeys ||
-      data.rbacPermissions.canWriteActivationKeys);
+  const { isLoading, isFetching, isSuccess, isError } = useUser();
+  const { hasAccess: hasAnyPermission } = usePermissionsWithContext([
+    'config-manager:activation_keys:*',
```

Comment:
This is not equal. This would basically mean the "hasAllActivationKeysPermissions", not if the user has any of them. To check if the user has at least one permission, I would try this:

```suggestion
    'config-manager:activation_keys:read', 'config-manager:activation_keys:write'
```

and also provide checkAll flag set to false: https://github.com/RedHatInsights/frontend-components/blob/8f4011ab00f9a4d01c564d4a9f3cd34d8bb09c5c/packages/utils/src/RBACHook/RBACHook.ts#L44C3-L44C11

Reply:
Hmm `checkAll` defaults to `false`? Why should it be explicitly set to false?

Reply:
I fixed it --
`'config-manager:activation_keys:read', 'config-manager:activation_keys:write'` would also require having BOTH the permissions not EITHER -- so I broke it down into two hooks